docs/docs/features/keymaps.mdx

```diff
@@ -121,18 +117,49 @@ Nested under the devicetree root, is the keymap node. The node _name_ itself is
 
 ### Layers
 
-Each layer of your keymap will be nested under the keymap node. Here is a sample
-that defines just one layer for this keymap:
+Each layer of your keymap will be nested under the keymap node. Here is an example of a layer in a 6-key macropad.
 
-<KeymapExample />
+```dts
+    keymap {
+        compatible = "zmk,keymap";
+
+        default_layer { // Layer 0
+// ----------------------------------------------
+// |     Z     |     M     |     K     |
+// |     A     |     B     |     C     |
+            bindings = <
+                &kp Z    &kp M    &kp K
+                &kp A    &kp B    &kp C
+            >;
+        };
+    }
+```
 
 Each layer should have:
 
 1. A `bindings` property this will be a list of [behavior bindings](../behaviors/index.mdx), one for each key position for the keyboard.
 1. (Optional) A `sensor-bindings` property that will be a list of behavior bindings for each sensor on the keyboard. (Currently, only encoders are supported as sensor hardware, but in the future devices like trackpoints would be supported the same way)
 
+### Multiple Layers
+
+Layers are numbered in the order that they appear in keymap node - the first layer is `0`, the second layer is `1`, etc.
+
+Here is an example of a trio of layers for a simple 6-key macropad:
+
+<KeymapExample />
+
+:::note
+Even if layer `1` was to be activated after `2`, layer `2` would still have priority as it is higher valued. Behaviors such as [`&to`](../behaviors/layers.md#to-layer) can be used to enable one layer _and disable all other non-default layers_, though.
```

Comment:
```suggestion
Even if layer `1` was to be activated after `2`, layer `2` would still have priority as it is higher valued. Behaviors such as [To Layer (`&to`)](../behaviors/layers.md#to-layer) can be used to enable one layer _and disable all other non-default layers_, though.
```
I am trying to be aware of future Studio users, where it isn't planned to surface the binding labels to the users at all. (We probably will need a lot more changes for that!)